templates/server-job.yaml

```diff
@@ -35,6 +35,10 @@ spec:
         app.kubernetes.io/version: {{ .Chart.AppVersion | replace "+" "_" }}
         app.kubernetes.io/component: database
         app.kubernetes.io/part-of: {{ .Chart.Name }}
+      annotations:
+        {{- with .Values.serviceJob.podAnnotations }}
```

Comment:
What is `serviceJob` here ?

Reply:
"serviceJob" variable refer to properties of "templates/server-job.yaml" file.